src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersStatusByEmailTest.java

```diff
@@ -128,14 +142,19 @@ public void rdcc_719_ac2_caseworker_publiclaw_courtadmin_role_should_return_403_
         userRoles.add("pui-organisation-manager");
         NewUserCreationRequest userCreationRequest = createUserRequest(userRoles);
 
-        // creating user in idam with the same email used in the invite user so that status automatically will update in the up
-        professionalApiClient.getMultipleAuthHeadersExternal(puiOrgManager, userCreationRequest.getFirstName(), userCreationRequest.getLastName(), userCreationRequest.getEmail());
+        // creating user in idam with the same email used in the invite user so that status automatically will update
+        // in the up

```

Comment:
This comment would read better if it was split earlier, suggestion:

```
        // creating user in idam with the same email used in the invite user
        // so that status automatically will update in the up
```